src/app/user-set-card/user-set-card.component.scss

```diff
@@ -0,0 +1,41 @@
+.user-sets-card{
+  display: grid;
+  grid-template-columns: min-content min-content min-content auto min-content;
+  width:99.5%;
+  height:67px;
+  box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+  padding-left: 16px;
+  padding-right: 16px;
+  align-items: center;
+  margin-bottom: 12px;
+  margin-left: 4px;
+}
+
+.user-sets-card mat-card-title{
+  line-height: 1em;
+  text-wrap:nowrap;
+}
+
+.study-set-badge{
+  width:fit-content;
+  float: left;
+  margin-left:12px;
+  border-radius: 8px;
+  font-weight: bold;
```

Comment:
The Figma design doesn't have the font bolded. I prefer the unbolded but I can understand why it would be bold as well. This might be a question for someone with a better eye for design. I am putting this here so we can bring it up at our meeting or have our supreme and gracious leader (@B-Yamawaki) decide. 
![image](https://github.com/PonderTeam/ponder-app/assets/4361319/afce5075-dc6b-48b3-87a7-110b531aff26)